### PR TITLE
Add examples for network_profile resource

### PR DESCRIPTION
**Pull request by dmettem**
Signed-off-by: Deepak Mettem <dmettem@vmware.com>

### CODE REVIEW

File: examples/network_profile/README.md [hunk 1]
```diff
@@ -7,3 +7,7 @@ This directory covers examples on how to create different kinds of Network Profi
 Follow these examples for setting up Network Profiles:
 
 * Setup [network_profile](../network_profile/simple/README.md) with no isolation
```
Comment (markpeek) [inline examples/network_profile/README.md]: Oops, perhaps I introduced this error. The relative path from this file to child directories should not include "../network_profile". This pertains to all of these new links.

File: examples/network_profile/isolation_with_external_subnet/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Getting Started
+
+There are variables which need to be added to terraform.tfvars. The first are for connecting to the vRealize Automation (vRA) endpoint. The others are names of cloud_account, and region that already setup in vRA.
+
+* `url` - The URL for the vRealize Automation (vRA) endpoint
+* `refresh_token` - The refresh token for the vRA account
+* `cloud_account` - The name of the cloud account added in vRA
+* `region` - The region within the cloud account
+* `subnet-name` - The subnet to add to Network Profile
```
Comment (markpeek) [inline examples/network_profile/isolation_with_external_subnet/README.md]: This is across all of these examples, we should standardize using underscores for all variables and not use dashes. I found this interesting best practice site: https://www.terraform-best-practices.com/naming

File: examples/network_profile/isolation_with_external_subnet/main.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
+provider "vra" {
+  url           = var.url
+  refresh_token = var.refresh_token
+}
+
+data "vra_cloud_account_aws" aws-cloud-account {
+       name = var.cloud_account
+}
+
+data "vra_region" "my-region" {
```
Comment (markpeek) [inline examples/network_profile/isolation_with_external_subnet/main.tf]: This is a general comment and something I likely introduced originally. We should think about removing the use of "my" in these examples and introduce a new variable (var.name?) that gets set in the invocation. That would make it easier to have multiple people use these as templates without conflicts with existing infrastructure. Thoughts?

File: examples/network_profile/README.md (hunk 1 above)
Comment (dmettem) [reply examples/network_profile/README.md]: You are right. They are not necessary. But not breaking the navigation as well. I removed the prefix from all the links this file.

File: examples/network_profile/isolation_with_external_subnet/main.tf (hunk 3 above)
Comment (dmettem) [reply examples/network_profile/isolation_with_external_subnet/main.tf]: I'm all up for removing "my" from the examples. But not sure if it is really necessary to introduce another variable. By "conflicts with existing infrastructure", do you mean the actual infrastructure that is being managed by these manifests or the resource/data-source blocks in the manifests?

File: examples/network_profile/isolation_with_external_subnet/README.md (hunk 2 above)
Comment (dmettem) [reply examples/network_profile/isolation_with_external_subnet/README.md]: Thanks for sharing the best practice site. I'm working on replacing all the dashes with underscores from these examples. I will update the PR soon after I'm done.

**Pull request merged**